sdk/python/kfp/compiler/compiler.py

```diff
@@ -549,7 +432,7 @@ def _compile(self, pipeline_func):
     for op in p.ops.values():
       sanitized_name = K8sHelper.sanitize_k8s_name(op.name)
       op.name = sanitized_name
-      for param in op.inputs + op.argument_inputs:
+      for param in op.inputs: # + op.argument_inputs:
```

Comment:
Do you still need this?

Reply:
No. I think I missed this when resolving the merge conflict.